packages/desktop/src/renderer/components/ChangeUsername/ChangeUsername.component.tsx

```diff
@@ -16,6 +16,7 @@ import { TextInput } from '../../forms/components/textInput'
 import { userNameField } from '../../forms/fields/createUserFields'
 
 import { parseName } from '@quiet/common'
+import { defaultLogger } from '../../logger'
```

Comment:
Could we just use 'logger' instead of 'defaultLogger'? I think the fact that the logger is 'default' should be implicit.

Reply:
This is a placeholder.  Eventually I'd like to update this to have named loggers like we have in every other module (e.g. `desktop:renderer:someModule`) but for now this is the default.